frontend/src/scenes/billing/ExportsUnsubscribeTable/exportsUnsubscribeTableLogic.tsx

```diff
@@ -0,0 +1,126 @@
+import { actions, afterMount, connect, kea, path, selectors } from 'kea'
```

Comment:
This file and `frontend/src/scenes/pipeline/ExportsUnsubscribeTable/exportsUnsubscribeTableLogic.tsx` seem to have identical code. Do you need both?